cmd/workspace/workspace/overrides.go

```diff
@@ -19,7 +27,43 @@ func exportOverride(exportCmd *cobra.Command, exportReq *workspace.ExportRequest
 	exportCmd.Annotations["template"] = `{{.Content | b64_decode}}`
 }
 
+// Give better errors / hints for common API errors.
+func wrapImportAPIErrors(err error, importReq *workspace.Import) error {
+	apiErr := &apierr.APIError{}
+	if !errors.As(err, &apiErr) {
```

Comment:
It seems like in the end it checks that the error type is `**apierr.APIError` instead of `apierr.APIError` or `*apierr.APIError`, is it expected?

Reply:
This checks that the error type is `*apierr.APIErr`. The `errors.As` method requires a pointer to a type that implements `error` (ie the `Error()` method)

In the Go SDK this is done by `*apierr.API` hence the double pointer being passed. see: https://github.com/databricks/databricks-sdk-go/blob/410c4b0a4cd0c38e97d8e5e3de7ddc2785fd9d12/apierr/errors.go#L65